sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/test/Dummy.groovy

```diff
@@ -1,4 +1,4 @@
-package life.qbic.business
```

Comment:
This class can be removed now since there is content in the sample-status-overview-domain and sample-status-overview-app modules

Reply:
This is no class but a package. I dont think I should remove that. What do you mean specifically? Can you elaborate?